addons/skin.estuary/xml/DialogVideoInfo.xml

```diff
@@ -382,10 +382,16 @@
 					<align>center</align>
 					<orientation>horizontal</orientation>
 					<scrolltime tween="quadratic">200</scrolltime>
-					<include content="InfoDialogButton">
+					<include content="InfoDialogButton" condition="String.IsEqual(ListItem.DBType,tvshow) | String.IsEqual(ListItem.DBType,season)">
+						<param name="id" value="8" />
+						<param name="icon" value="icons/filemanager.png" />
+						<param name="label" value="$VAR[VideoInfoPlayLabelVar]" />
```

Comment:
Why using a $VAR[] here when you already check the condition for the include?

Reply:
The intention was to set both the button's label and the button's icon using var's so that only one control / include was needed. Unfortunately the icon path could not be set through a var in a param no matter what I tried. I believe I came across something about limitations of var's in a param on the forum at some point, but couldn't find it. 
Ideally, that shortcoming would be solved and the include can be dropped here.

Reply:
I am not sure if you understood correctly :)
The first button you created is only included for tvshows and seasons. Why do you also check that again in a new VAR instead of directly assigning label 1024 to that button?

Reply:
I understood. Ideally though only one control include would be needed. But as this currently doesn't seem possible (see my previous comment), I've removed the var.

Reply:
i think it is possible by using the include InfoDialogToggleButton. Did you try that?